### ISSUE TITLE
Set default value to ... Error

### ISSUE THREAD

**Issue by Bretto** [orientechnologies/orientdb#3908]
http://www.orientechnologies.com/docs/last/SQL-Alter-Property.html

```
alter property Client.created default sysdate()
```
```
Error: com.orientechnologies.orient.core.sql.OCommandSQLParsingException: Error on parsing command at position #27: Unknown property attribute 'DEFAULT'. Supported attributes are: [LINKEDTYPE, LINKEDCLASS, MIN, MAX, MANDATORY, NAME, NOTNULL, REGEXP, TYPE, CUSTO
M, READONLY, COLLATE]
Command: alter property User.created default sysdate()
```

**Comment by lvca** [comment]
Release?

**Comment by Bretto** [comment]
I am running 2.0.6, I see now its only available on 2.1... Thanks for the feedback.
